packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -149,17 +226,37 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
     return _tokens;
   }
 
+  function getOtherReserveAddresses() external view returns (address[] memory) {
```

Comment:
These getters aren't strictly necessary since the variables are public

Reply:
Ok. Public array getters need associated array length getters, though. I replaced 3 array getting functions with 2 array length getting functions, since two of the arrays are constrained to have equal lengths.

Reply:
Oh, maybe your original solution was better then..